hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/MonoTransactionDispatcher.java

```diff
@@ -95,6 +95,7 @@ public void dispatchHandle(@NonNull final HandleContext context) {
             case TOKEN_FEE_SCHEDULE_UPDATE -> dispatchTokenFeeScheduleUpdate(context);
             case TOKEN_DELETION -> dispatchTokenDeletion(context);
             case TOKEN_BURN -> dispatchTokenBurn(context);
+            case TOKEN_WIPE -> dispatchTokenWipe(context);
```

Comment:
I wonder if we can stop to maintain `MonoTransactionDispatcher`. Are we using this in some of the integration tests?

Reply:
Wondered about the same. It was used only for using `workflows.enabled`. I don't think this is needed anymore.

Reply:
Ok, I won't write anything more in `MonoTransactionDispatcher` :)